### ISSUE TITLE
"fit" doesn't work with jasmine 2.x on protractor

### ISSUE THREAD

**Issue by joaomqcunha** [larrymyers/jasmine-reporters#87]
I'm using the `protractor@1.8.0` with `jasmine@2.0` and using `jasmine-reporters@2.0.4`.

When I set an `it` as `fit`, I have the following message:

```javascript
Running "protractor:smoke" (protractor) task
Starting selenium standalone server...
[launcher] Running 1 instances of WebDriver
Selenium standalone server started at http://192.168.1.3:57112/wd/hub
Spec started

Executed 0 of 10 specs SUCCESS (10 SKIPPED) in 0.002 secs.
```

**0 tests run!**

Although, when I use "fdescribe", it works as expected.

I first opened an issue on protractor github project but we concluded that the problem most likely be on `jasmine-reporters` (https://github.com/angular/protractor/issues/1908)

Can you guys help me with this?

**Comment by bloveridge** [comment]
This is strange. I've confirmed the behavior, and it's not just the JUnit reporter -- TerminalReporter, at least, suffers from the same problem, which means all of the 2.x reporters could be similarly broken. I've not tracked it down yet, but it's something happening in specStarted / specDone callbacks. I'll keep digging and update all reporters once I find the problem.

Thanks for raising the alarm.

**Comment by bloveridge** [comment]
This should be fixed in 2.0.5, which was just released.

**Comment by joaomqcunha** [comment]
You welcome @bloveridge! Thank you for this fast resolution :+1:
